docs/en/Tiered-Storage-on-Alluxio.md

```diff
@@ -1,88 +1,117 @@
+buffer cache. This fast data layer between user applications and the various under storages results
+in vastly improved I/O performance.
 
-* MEM (Memory)
-* SSD (Solid State Drives)
-* HDD (Hard Disk Drives)
+The amount and type of storage for each Alluxio node to manage is determined by user configuration.
+Alluxio also supports tiered storage, which makes the system storage media aware, enabling data
+storage optimizations similar to L1/L2 cpu caches.
 
-Using Alluxio with tiered storage allows Alluxio to store more data in the system at once,
-since memory capacity may be limited in some deployments. With tiered storage, Alluxio
-automatically manages blocks between all the configured tiers, so users and administrators do not
-have to manually manage the locations of the data. Users may specify their own data management
-strategies by implementing [allocators](#allocators) and [evictors](#evictors). In addition, manual
-control over tier storage is possible, see [pinning files](#pinning-files).
+## Configuring Alluxio Storage
 
-## Using Tiered Storage
+The easiest way to configure Alluxio storage is to use the default single-tier mode.
 
-With the introduction of tiers, the data blocks managed by Alluxio are not necessarily in memory;
-blocks can be in any of the available tiers. To manage the placement and movement of the blocks,
-Alluxio uses *allocators* and *evictors* to place and re-arrange blocks between the tiers. Alluxio
-assumes that tiers are ordered from top to bottom based on I/O performance. Therefore, the typical
-tiered storage configuration defines the top tier to be MEM, followed by SSD, and finally HDD.
+Out-of-the-box Alluxio will provision a ramdisk on every worker and take a percentage of the
+system's total memory. This ramdisk will be used as the only storage medium allocated to each
+Alluxio worker.
 
-### Storage Directories
+A common modification to the default is to explicitly set the ramdisk size. For example, to set the
+ramdisk size to be 16GB on each worker:
 
-A tier is made up of at least one storage directory. This directory is a file path where the Alluxio
-blocks should be stored. Alluxio supports configuring multiple directories for a single tier,
-allowing multiple mount points or storage devices for a particular tier. For example, if you have
-five SSD devices on your Alluxio worker, you can configure Alluxio to use all five devices for the
-SSD tier. Configuration for this is described [below](#enabling-and-configuring-tiered-storage).
-Choosing which directory the data should placed is determined by the [allocators](#allocators).
+```
+alluxio.worker.memory.size=16GB
+```
 
-### Writing Data
+Another common change is to specify multiple storage media, such as ramdisk and SSDs. We will need
+to update `alluxio.worker.tieredstore.level0.dirs.path` to take specify each storage medium we want
+to use as a storage directory. For example, to use the ramdisk (mounted at /mnt/ramdisk) and 2 ssds
```

Comment:
nit: `` `/mnt/ramdisk` ``

Reply:
also, capitalize "SSD", to stay consistent with other parts.

Reply:
Updated, thanks.